rudder-agent/SOURCES/systemd/cf-serverd.service

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=CFEngine file server
+After=syslog.target
+After=network.target
+ConditionPathExists=/var/rudder/cfengine-community/bin/cf-serverd
+ConditionPathExists=/var/rudder/cfengine-community/inputs/promises.cf
+ConditionPathExists=!/opt/rudder/etc/disable-agent
+ReloadPropagatedFrom=rudder-agent.service
+PartOf=rudder-agent.service
```

Comment:
Shouldn't this be removed to enable disabling cf-execd without disabling rudder-agent ?